test/unit/language-test.js

```diff
@@ -309,6 +309,10 @@ describe('languageUtils', function() {
             };
         });
 
+        after(function () {
```

Comment:
I would move your unit test to it's own `describe` block. Make one for `forceLocalizationDefaults`